modules/docs/src/docs/user-guide/30_modules/20_http-client.adoc

```diff
@@ -53,3 +53,23 @@ val client: Client[F] = pillars.httpClient
 
 This `Client[F]` instance can be used to send HTTP requests by using the same methods as `org.http4s.client.Client[F]`.
 
+In addition to `org.http4s.client.Client[F]` methods, the `HttpClient` module provides methods to directly call a tapir endpoint:
+
+* `call`: Calls a tapir public endpoint with the specified input and returns either the output or the error output.
+* `callSecure`: Calls a tapir secure endpoint with the specified input and security input and returns either the output or the error output.
```

Comment:
```suggestion
* `call`: Calls a tapir public endpoint with the specified input and returns either the output or the error output.
* `callSafe`: Calls a tapir secure endpoint with the specified input and security input and returns either the output or the error output.
```

Reply:
I'm not sure about the `callSafe` naming as tapir uses the word secure in `SecureRequest` and in the docs